test/dygraph_to_static/test_lac.py

```diff
@@ -66,9 +65,9 @@ def __init__(
         self.is_reverse = is_reverse
 
     def forward(self, inputs):
-        # Use `to_variable` to create a copy of global h_0 created not in `DynamicGRU`,
+        # Use `paddle.to_tensor` to create a copy of global h_0 created not in `DynamicGRU`,
         # to avoid modify it because `h_0` is both used in other `DynamicGRU`.
-        hidden = to_variable(self.h_0)
+        hidden = paddle.to_tensor(self.h_0)
```

Comment:
```suggestion
        hidden = paddle.assign(self.h_0)
```

根据注释，这里替换成 `paddle.assign`，`paddle.to_tensor` 在已经是 Tensor 的情况下不会创建 copy